File: proyecto-compose-android/app/src/main/java/mx/ipn/escom/trejol/proyectocomposemoviles/services/models/orders/DeliveryLocation.kt
package mx.ipn.escom.trejol.proyectocomposemoviles.services.models.orders

data class DeliveryLocation(
    val address: String,
    val city: String,
    val cp: String,
    val created_at: String,
    val id: Int,
    val is_default: Int,
    val name: String,
    val state: String,
    val status: Int,
    val updated_at: String,
    val user_id: Int
)

File: pedidos-api/app/Models/DeliveryLocation.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class DeliveryLocation extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'address',
        'city',
        'state',
        'cp',
        'user_id'
    ];
}
